### DISCUSSION TITLE
Support Product Page Optimization (iOS 15)

### DISCUSSION THREAD

**Discussion by joeboyscout04** [fastlane/fastlane#19368] [Ideas]
With iOS 15 comes the ability to do "Product Page Optimization" on the App Store.  

https://developer.apple.com/app-store/product-page-updates/

I'm not sure if this is even supported by the AppstoreConnect API, but would be a nice feature to add to Fastlane if/when it's supported.

**Comment by dinsen** [comment]
There is no new endpoints for it.
The only new thing is a endpoint for releasing apps that are Pending Developer Release" and endpoints for Xcode Cloud:
https://developer.apple.com/documentation/appstoreconnectapi?changes=latest_minor

**Reply by Huaba93** [reply]
How about this?
https://developer.apple.com/documentation/appstoreconnectapi/app_store_version_experiment_treatment_localizations?changes=latest_minor

**Comment by Robert9655** [comment]
That tickles the funny bone! At least here or if I try an article I can get away with a few more paragraphs!

[TellHappystar](https://www.tellhappystar.net/)

**Comment by ChaseTargaryen** [comment]
Will this be a feature in Fastlane?

**Comment by CraigSiemens** [comment]
It appears like there's been APIs added for this, it's be nice if fastlane could support it.

https://developer.apple.com/documentation/appstoreconnectapi/app_store/custom_product_pages_and_localizations/app_custom_product_pages

**Comment by Mykyta-Chernenko** [comment]
We automatize ASO for our products, and being able to do A/B tests is crucial for us. Bumping up this feature visibility 🥹
Thank you, you are great!
